tests/src/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -80,6 +80,7 @@ class ApiGwTests
       }
       finally {
         val deleteresult = wsk.api.delete(basepathOrApiName = testbasepath)
+        Thread.sleep(5000)  // Test suite will overrun the local dev throttling limit
```

Comment:
Yes that's another alternative to excempt whisk.system during testing, or your saying I general ?
I think having limits on whisk.system is also a protection we have in place, maybe increasing the limit on some priviledge namespaces would be a way to do if you are referring in general and not test run. 

There is no way to eliminate or set higher  limits and then in the specific throttle tests set the limits before running the tests dynamically and then when test is finish remove the limits?  

This way no other tests are affected by this lower limit set by default